sui/token_bridge/sources/state.move

```diff
@@ -134,14 +134,6 @@ module token_bridge::state {
         self: &mut State,
         implementation_digest: Bytes32
     ): UpgradeTicket {
```

Comment:
since it's not obvious that the version check is performed by the runtime, we should include a comment here that states that explicitly, to save future readers a minor heart attack

Reply:
Added a note just above `authorize_upgrade` and `commit_upgrade` in both Wormhole and Token Bridge. How does it look?